nautobot/dcim/filters/__init__.py

```diff
@@ -1628,6 +1629,11 @@ class PowerPanelFilterSet(LocatableModelFilterSetMixin, NautobotFilterSet):
         to_field_name="name",
         label="Rack group (name or ID)",
     )
+    panel_type = django_filters.MultipleChoiceFilter(
+        choices=PowerPanelTypeChoices,
+        null_value=None,
+    )
+    breaker_position_count = django_filters.NumberFilter()
```

Comment:
Please add these new filters to the `PowerFeedTestCase.generic_filter_tests` in `dcim/tests/test_filters.py`.